pages/irc.php

```diff
@@ -1,14 +1,22 @@
 <?php
 $title = __("IRC Page");
+if(!$loguserid)
+{
+  Kill("You should be log in for use the IRC.");
+}
+else
+{
+$nick = "insert default guest nick here";

```

Comment:
No. If you require logging in (for absolutely no reason) then this is completely stupid.